src/util/system.h

```diff
@@ -271,11 +271,18 @@ class ArgsManager
     /**
      * Get data directory path
      *
-     * @param net_specific Append network identifier to the returned path
      * @return Absolute path on success, otherwise an empty path when a non-directory path would be returned
      * @post Returned directory path is created unless it is empty
      */
-    const fs::path& GetDataDirPath(bool net_specific = true) const;
+    const fs::path& GetDataDirBase() const { return GetDataDir(false); }
```

Comment:
80dec1717258fd74ae32274affed71daf2949e46: The goal of this pull is to remove the `GetDataDir` function, but here you introduce more call sites?

Also, it could make sense to have this in a separate commit and move the other mechanical changes to a scripted diff

Reply:
> [80dec17](https://github.com/bitcoin/bitcoin/commit/80dec1717258fd74ae32274affed71daf2949e46): The goal of this pull is to remove the `GetDataDir` function, but here you introduce more call sites?

`GetDataDir(false)` actually calls `const fs::path& ArgsManager::GetDataDir(bool net_specific) const` (i.e. https://github.com/bitcoin/bitcoin/commit/80dec1717258fd74ae32274affed71daf2949e46#diff-a5795f7946088222aa59a662e15f5b5aee627bfe2311459e789d2a46783e37cbR455). But I can see that it is very actually very confusing. 

I'll try to go with https://github.com/bitcoin/bitcoin/pull/21850#discussion_r637092957 now and hopefully it will be nicer.